src/api/transfers/routes.js

```diff
@@ -91,7 +91,7 @@ module.exports = [{
     },
     validate: {
       headers: Joi.object({
-        'content-type': Joi.string().required().valid('application/json'),
+        'content-type': Joi.string().required().valid('application/vnd.interoperability.transfers+json;version=1.0'),
```

Comment:
Why don’t you have an ‘accept’ here?

Reply:
regarding 'accept'.. that is new requirement, new task is added to the story, so Sam asked me to include it as part of this PR

Reply:
Accept header is applicable only for POST and GET (and DELETE) calls as per the Spec and not for PUT, so not needed here.